tests/karma/search/containers/TestAddonPage.js

```diff
@@ -19,20 +47,150 @@ describe('AddonPage', () => {
     ));
   }
 
-  it('renders the name', () => {
+  describe('rendered fields', () => {
     const root = render({state: initialState, props: {params: {slug: 'my-addon'}}});
-    assert.equal(root.querySelector('h1').textContent, 'Addon!');
+
+    it('renders the name', () => {
+      assert.equal(root.querySelector('h1').textContent, 'Addon!');
+    });
+
+    it('renders the summary', () => {
+      assert.equal(root.querySelector('.addon--summary').textContent, 'My add-on');
+    });
+
+    it('renders the description', () => {
+      assert.equal(
+        root.querySelector('.addon--description').textContent, 'An add-on that adds on.');
+    });
+
+    it('renders the tags', () => {
+      const tags = root.querySelector('.addon--tags').childNodes;
```

Comment:
They're `li`s. I used the child nodes because I didn't want to tie it to a certain implementation. They'll likely either be `li`s in a `ul` or `div`s in a container, either way it shouldn't matter (as long as no one adds any more markup I guess).